src/OSPSuite.Core/Domain/UnitSystem/DimensionFactory.cs

```diff
@@ -124,6 +124,11 @@ public IDimension DimensionForUnit(string unitName)
          return matches.FirstOrDefault(x => x.Units.Any(unit => string.Equals(unitName, unit.Name, StringComparison.Ordinal))) ?? matches.First();
       }
 
+      public IDimension CreateUserDefinedDimension(string dimensionName, string unit)
```

Comment:
Theoretically, we will export PK PArameter in dimension that are unknown. Therefore I moved this definition in the dimensionFactory so that we can import object in an unknown dimension. This should only be used from R. But this might show up in PKSim when importing results